decoder/ihevcd_sao.c

```diff
@@ -1114,6 +1115,8 @@ void ihevcd_sao_shift_ctb(sao_ctxt_t *ps_sao_ctxt)
             ps_codec->s_func_selector.ihevc_memcpy_fptr(pu1_src_top_luma, &pu1_src_luma[(sao_ht_luma - 1) * src_strd], sao_wd_luma);
         }
 
+        if (CHROMA_FMT_IDC_MONOCHROME != ps_sps->i1_chroma_format_idc)
```

Comment:
nit: can this condition be combined with the check below, so that indentation is still properly maintained